client/i18n/locales/english/intro.json

```diff
@@ -3480,7 +3480,12 @@
           "In these lecture videos, you will learn about working with forms in React."
         ]
       },
-      "sgau": { "title": "276", "intro": [] },
+      "workshop-react-forms": {
+        "title": "Build a Superhero Application Form",
+        "intro": [
+          "In this workshop, you'll learn how to work with forms in React by building a superhero application form."
```

Comment:
For previous lab and workshops we avoided to use "you'll learn" and used "practice" or something similar.